tree/dataframe/src/RSqliteDS.cxx

```diff
@@ -60,11 +345,24 @@ constexpr char const *RSqliteDS::fgTypeNames[];
 RSqliteDS::RSqliteDS(std::string_view fileName, std::string_view query)
    : fDb(nullptr), fQuery(nullptr), fNSlots(0), fNRow(0)
 {
+   static bool isRegistered = Register();
```

Comment:
the `static` here means that only the first `RSqliteDS` constructor call will try to register...if that fails, all subsequent ctor calls will not even try to register and just throw. does it not make sense to have a retry mechanism?

Reply:
I think we wouldn't gain from a retry. The registration is only setting the function pointers. It always works iff Davix is available. For the actual HTTP operations, Davix implements a retry logic.

Reply:
I see, but then this is not really `isRegistered` but rather `isDavixAvailable`..?